airflow/example_dags/example_dataset_alias.py

```diff
@@ -22,15 +22,13 @@
 
 Turn on all the DAGs.
 
-Before running any DAG, the schedule of the "dataset-alias-consumer" DAG will show as "Unresolved DatasetAlias".
+Before running any DAG, the schedule of the "dataset_alias_example_alias_consumer_with_no_taskflow" DAG will show as "unresolved DatasetAlias".
```

Comment:
This should be `dataset_alias_example_alias_consumer`? The `no_taskflow` DAG is in another file.

Reply:
Yes @uranusjr. fixing it